coda-prototype/src/ui/editorUI.js

```diff
@@ -254,11 +254,8 @@ var codeEditorManager = {
                     .removeClass("has-error");
 
                 newDataset.schemes[newId] = tempScheme;
-                messageViewerManager.codeSchemeOrder = [newId + ""].concat(messageViewerManager.codeSchemeOrder);
-                activeSchemeId = newId + "";
-                messageViewerManager.activeScheme = newId + "";
```

Comment:
This and the previous line were redundant because messageViewerManager.addNewSchemeColumn will set these.